seqr/management/commands/transfer_families_to_different_project.py

```diff
@@ -1,17 +1,21 @@
 from django.core.management.base import BaseCommand
 
-from seqr.models import Project, Family, VariantTag, VariantTagType
+from seqr.models import Project, Family, VariantTag, VariantTagType, Sample
 from seqr.utils.search.utils import backend_specific_call
 
 import logging
 logger = logging.getLogger(__name__)
 
 
-def _validate_no_search_families(families):
-    search_families = families.filter(individual__sample__is_active=True).distinct().values_list('family_id', flat=True)
-    if search_families:
-        logger.info(f'Unable to transfer the following families with loaded search data: {", ".join(search_families)}')
-    return families.exclude(individual__sample__is_active=True)
+def _disable_search(families):
+    search_samples = Sample.objects.filter(is_active=True, individual__family__in=families)
+    if search_samples:
+        updated_families = search_samples.values_list("individual__family__family_id", flat=True).distinct()
+        family_summary = ", ".join(sorted(updated_families))
+        num_updated = search_samples.update(is_active=False)
```

Comment:
There was some talk about these samples not counting towards the global allele count, I'm guessing there's code somewhere that only counts samples where `is_active` is false. Would you mind pointing me to that code?

Reply:
that is unrelated to seqr code - all seqr can do is remove the ability to perform searches on these families. Ben will need to do a separate manual action to actually remove the data from the backend